src/Compute/Compute/Manual/VirtualMachineScaleSetCreateOrUpdateMethod.cs

```diff
@@ -569,7 +569,7 @@ private async Task<ResourceConfig<VirtualMachineScaleSet>> SimpleParameterSetOrc
                     instanceCount: _cmdlet.InstanceCount,
                     dataDisks: _cmdlet.DataDiskSizeInGb,
                     zones: _cmdlet.Zone,
-                    ultraSSDEnabled: _cmdlet.EnableUltraSSD.IsPresent,
+                    ultraSSDEnabled: (_cmdlet.EnableUltraSSD == true) ? true : (bool?)null,
```

Comment:
Maybe just pass the value? If the user doesn't have access to that feature, then they shouldn't be able to access the feature related parameter, right? Check that. If that is true, then can just pass in the value as opposed to explicitly null. 
Wait, no the switch param defaults to False if not set, right? So cannot pass that into the property without worry for feature flag restriction.